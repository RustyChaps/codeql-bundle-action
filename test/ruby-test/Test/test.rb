class Source
    def source
        "untrusted input"
    end 
end

source = Source()
source.source()