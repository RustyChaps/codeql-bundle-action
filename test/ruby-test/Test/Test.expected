| test.rb:8:1:8:15 | call to source |
